ansible_events_ui/api/activation.py

```diff
@@ -1,8 +1,43 @@
 """Activation API endpoints."""
-from fastapi import APIRouter
+import logging
+
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from fastapi.exceptions import HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ansible_events_ui import schemas
+from ansible_events_ui.db import models
+from ansible_events_ui.db.dependency import get_db_session
+
+logger = logging.getLogger("ansible_events_ui")
 
 __all__ = ("router",)
 
 router = APIRouter()
 
-# Implement activation API here.
+
+@router.post("/api/activations/", response_model=schemas.Activation)
```

Comment:
Could you add the `operation_id` param for the openapi spec? 
```suggestion
@router.post("/api/activations/", response_model=schemas.Activation, operation_id="create_activation")
```